src/arrayMethodJoin.js

```diff
@@ -4,8 +4,20 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let joinedArray = '';
+
+    for (const word of this) {
+      if (word != null && word !== undefined) {
+        joinedArray += word;
+      }
+
+      if (this.indexOf(word) !== this.length - 1) {
```

Comment:
Something is wrong here. Try to fix this(When I use your method on the next array - [0, 1, 2, 3, 1])
<img width="331" alt="image" src="https://github.com/mate-academy/js_array-method-join/assets/86789161/0185ef60-0e94-4df1-a129-a2f76b275d1a">

I got ',' at the end of the row. It's not what is expected
<img width="87" alt="image" src="https://github.com/mate-academy/js_array-method-join/assets/86789161/b5d3c3af-851a-4ee9-ba54-108a2ed43af7">